src/proc_softirqs.c

```diff
@@ -144,26 +170,27 @@ int do_proc_softirqs(int update_every, unsigned long long dt) {
                 for(l = 0; l < lines ;l++) {
                     struct interrupt *irr = irrindex(irrs, l, cpus);
                     if(unlikely(!irr->used)) continue;
-                    core_sum += irr->value[c];
+                    core_sum += irr->cpu[c].value;
                 }
                 if(unlikely(core_sum == 0)) continue; // try next core
```

Comment:
Unrelated, but why is this test only in `src/proc_softirqs` and not in `src/proc_interrupts`?